src/networking/responses/products-response.ts

```diff
@@ -11,6 +11,7 @@ export interface PricingPhaseResponse {
 
 export interface PurchaseOptionResponse {
   id: string;
+  price_id: string;
```

Comment:
This is theoretically optional in the backend response, but it's never optional in practice, I'll double check if I can make it mandatory later on.